docs/sources/flow/reference/components/loki.source.kubernetes.md

```diff
@@ -159,6 +161,19 @@ arguments.
 
 [using clustering]: {{< relref "../../concepts/clustering.md" >}}
 
+### default_instance_label block
+
+The `default_instance_label` block configures default "instance" label to be assigned with logs 
+scraped from inputted targets.
```

Comment:
I'm not really following what this is describing. If the `enabled` argument is set to `true` in the `default_instance_label` block then you can explicitly assign an instance label with `value` argument? 

So, how does this relate to "logs scraped from inputted targets"? Is this what happens when you don't set `enabled` to `true`?

Reply:
>  If the enabled argument is set to true in the default_instance_label block then you can explicitly assign an instance label with value argument?

yes, this is what I'm trying to convey

> So, how does this relate to "logs scraped from inputted targets"? Is this what happens when you don't set enabled to true?
The scraped logs from inputted targets are the pod logs to have the default "instance" label. If we don't set `enabled=true`, the behavior would stay the same as before, i.e. automatically assign an "instance" label

Reply:
```suggestion
You can use the `default_instance_label` block to explicitly define the default "instance" label assigned to the logs 
scraped from the targets.
```

So, something like this?

Reply:
Yes! That's the spot